apis/r/R/SOMASparseNDArray.R

```diff
@@ -51,8 +51,8 @@ SOMASparseNDArray <- R6::R6Class(
                      config = cfg,
                      dim_points = coords,
                      result_order = result_order,
-                     timestamprange = if (is.null(private$tiledb_timestamp)) private$tiledb_timestamp
-                                      else c(0, private$tiledb_timestamp),
+                     timestamprange = if (is.null(self$tiledb_timestamp)) self$tiledb_timestamp
+                                      else c(0, self$tiledb_timestamp),
```

Comment:
Actually see the difference between this and the preceding block?  If we carry a two elem vector (or NULL) through we no longer need the if / else.  I just hadn't gotten to NDArray yet.  Likely cleanup?

Reply:
All I've done is change all calls to `private$tiledb_timestamp` to `self$tiledb_timetamp`, leaving the code exactly as it was from your branch

Reply:
Yeah. I was speaking half to myself here :)